src/functions/Coverage.ps1

```diff
@@ -1043,6 +1043,221 @@ function Get-JaCoCoReportXml {
+                $package.Line.Covered += $covered
+                $package.Line.Total += $total
+                $package.Classes.$file.Line.Missed += $missed
+                $package.Classes.$file.Line.Covered += $covered
+                $package.Classes.$file.Line.Total += $total
+                $package.Classes.$file.Methods.$function.Line.Missed += $missed
+                $package.Classes.$file.Methods.$function.Line.Covered += $covered
+                $package.Classes.$file.Methods.$function.Line.Total += $total
+                $package.Classes.$file.Lines.$line = @{
+                    Instruction = @{ Missed = 0; Covered = 0; Total = 0 }
+                }
+            }
+
+            $package.Instruction.Missed += $missed
+            $package.Instruction.Covered += $covered
+            $package.Instruction.Total += $total
+            $package.Classes.$file.Instruction.Missed += $missed
+            $package.Classes.$file.Instruction.Covered += $covered
+            $package.Classes.$file.Instruction.Total += $total
+            $package.Classes.$file.Methods.$function.Instruction.Missed += $missed
+            $package.Classes.$file.Methods.$function.Instruction.Covered += $covered
+            $package.Classes.$file.Methods.$function.Instruction.Total += $total
+            $package.Classes.$file.Lines.$line.Instruction.Missed += $missed
+            $package.Classes.$file.Lines.$line.Instruction.Covered += $covered
+            $package.Classes.$file.Lines.$line.Instruction.Total += $total
+        }
+
+        $report.Class.Missed += $package.Class.Missed
+        $report.Class.Covered += $package.Class.Covered
+        $report.Class.Total += $package.Class.Total
+        $report.Method.Missed += $package.Method.Missed
+        $report.Method.Covered += $package.Method.Covered
+        $report.Method.Total += $package.Method.Total
+        $report.Line.Missed += $package.Line.Missed
+        $report.Line.Covered += $package.Line.Covered
+        $report.Line.Total += $package.Line.Total
+        $report.Instruction.Missed += $package.Instruction.Missed
+        $report.Instruction.Covered += $package.Instruction.Covered
+        $report.Instruction.Total += $package.Instruction.Total
+
+        $packageList.Add($package)
+    }
+
+    $commonParent = Get-CommonParentPath -Path $CoverageReport.AnalyzedFiles
+    $commonParentLeaf = & $SafeCommands["Split-Path"] $commonParent -Leaf
+
+    # the Cobertura xml format without the doctype, as the XML stuff does not like DTD's.
+    $xmlDeclaration = '<?xml version="1.0" encoding="utf-8"?>'
+    $coberturaReport = $xmlDeclaration
+    $coberturaReport += '<coverage>'
+    $coberturaReport += "<sources><source>$commonParent</source></sources>"
+    $coberturaReport += '</coverage>'
+
+    [xml] $coberturaReportXml = $coberturaReport
+
+    $coverageElement = $coberturaReportXml.coverage
+    Add-XmlAttribute -Element $coverageElement -Attributes @{
```

Comment:
```suggestion
    Add-XmlAttribute -Element $coverageElement -Attributes [ordered]@{
```
Hashtables are sorted by hash which is pretty random. Making it an `OrderedDictionary` like this should make the attribute-ordering consistent